libbeat/common/field.go

```diff
@@ -31,12 +33,30 @@ type Field struct {
 	Dynamic        DynamicType `config:"dynamic"`
 	Index          *bool       `config:"index"`
 	DocValues      *bool       `config:"doc_values"`
+	Analyzed       *bool       `config:"analyzed"`
```

Comment:
Could you add a line here `// Kibana specific fields` and also move Pattern down here?